pontoon/contributors/templates/contributors/settings.html

```diff
@@ -33,6 +34,8 @@ <h2 id="username">
   {% if user.translated_locales %}
     {{ Checkbox.checkbox('Make suggestions', class='force-suggestions', attribute='force_suggestions', is_enabled=user.profile.force_suggestions, title='Save suggestions instead of translations') }}
   {% endif %}
+  
+  {{ LocaleHomepage.select(user.profile.custom_homepage, all_locales) }}
```

Comment:
Have you tried reusing [pontoon/teams/templates/teams/widgets/team_selector.html](#diff-a4ff7ce5b3f91c38b82a5e9fc30eb73c) here? It's basically the same selector, with additional "Default Pontoon Homepage" option. And a label.